DemoApp/DemoApp/ViewModels/TMChipsPageViewModel.cs

```diff
@@ -0,0 +1,46 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using DemoApp.Constant;
+using System.Collections.ObjectModel;
+using System.Windows.Input;
+using Trimble.Modus.Components;
+using Trimble.Modus.Components.Enums;
+
+namespace DemoApp.ViewModels
+{
+    internal partial class TMChipsPageViewModel : ObservableObject
+    {
+        private int count = 0;
+        public ObservableCollection<TMChips> ChipsCollection { get; } = new ObservableCollection<TMChips>();
+
+        public TMChipsPageViewModel()
+        {
+            ChipsCollection.Add(new TMChips() { Title = "Chip 1", Type = ChipType.Input, LeftIconSource = ImageConstants.AccountIcon, ChipsSize = ChipSize.Small });
```

Comment:
TMChip is a UI component. Shouldn't be here in the ViewModel. For this, create a DataModel class like a Person object or something. You can have the Title and the Image in that and you can hardcode the Type and the ChipsSize in the xaml as they are not changing

Reply:
Added a Chips Item for the collection